MTNeuro/self_supervised/transforms/xray.py

```diff
@@ -20,6 +20,7 @@ class XrayAugmentationPipeline(nn.Module):
     #todo: improve efficiency by removing conditionals from forward pass
     def __init__(self, transform_name_list, name=None, image_size=IMAGE_SIZE, p = 0.5, same_on_batch=False, dataset_type="2D"):
         super(XrayAugmentationPipeline, self).__init__()
+        self.resize = Resize((2*image_size, 2*image_size))
```

Comment:
@Jorge-Quesada Might this resize change anything with respect to reproducibility of paper results with this codebase?

Reply:
Hey @lakshmi-sathi no it doesn't, the image_size variable that is fed into the augmentation class is the target crop size (which is always half of the original image size), so that was there for the different downsampling cases (4x, 2x and fullscale). If it causes confusion we can actually remove it since now the resize is defined in the main scripts (this resize is no longer used)